tools/topology/topology2/include/components/google-rtc-aec.conf

```diff
@@ -47,6 +47,15 @@ Class.Widget."google-rtc-aec" {
 		token_ref	"comp.word"
 	}
 
+	Object.Control.bytes."1" {
+		access [
+			tlv_write
+			tlv_callback
+		]
+		name 'Config'
```

Comment:
@yongzhi1 can the name be changed to something specific like google-rtc-config instead?

Reply:
The generated name is 'google-rtc-aec.18.1 Config' right now, then 'google-rtc-aec.18.1 google-rtc-config' is a little too verbose?